eclass/kernel-build.eclass

```diff
@@ -29,6 +29,11 @@ if [[ ! ${_KERNEL_BUILD_ECLASS} ]]; then
 _KERNEL_BUILD_ECLASS=1
 
 PYTHON_COMPAT=( python3_{10..12} )
+if [[ ${KERNEL_IUSE_MODULES_SIGN} ]]; then
```

Comment:
I don't exactly like having two control vars that pretty much will always be used together but I guess at this point you'd have to rename the old one for this to make sense with just one var.

Reply:
Yeah me neither, I did it like this anyway because it feels a bit weird to use a var here that we have defined and documented in another eclass.

I'd be open to doing this differently though, but I don't really see a proper alternative.